test/modules/convert-array-of-arrays-to-csv.spec.js

```diff
@@ -62,3 +64,9 @@ test('convertArrayOfArraysToCSV | array of arrays with values of null and undefi
 
   expect(result).toBe(expectedResultArrayNullAndUndefined);
 });
+
+test('convertArrayOfArraysToCSV | array of arrays with value of zero | with default options and no header', () => {
```

Comment:
Just one little thing. 


could you change

```js
test('convertArrayOfArraysToCSV | array of arrays with value of zero | with default options and no header', () => {
```
to
```js
test('convertArrayOfArraysToCSV | array of arrays with value of zero | with default options and header', () => {
```

because you are actually testing it with header not without.